src/main/java/dk/cngroup/trainings/spring/springassignment/controller/rest/AnimalController.java

```diff
@@ -50,19 +51,22 @@ public AnimalController(AnimalService animalService) {
 
 	@RequestMapping(path = "/{id}", method = RequestMethod.DELETE)
 	public ResponseEntity<String> deleteAnimalById(@PathVariable("id") long id) {
-		if (animalService.deleteAnimalById(id)) {
+		try {
```

Comment:
In here, I think it was fine to use if/else (it makes sense for deleteAnimalById to return boolean), but this is also fine.